ironic/resource_ironic_node_v1.go

```diff
@@ -174,6 +183,25 @@ func resourceNodeV1() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"bmc_address": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"bmc_disable_certificate_verification": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
```

Comment:
I think these new fields duplicate data that already exists in the `driver_info` ?

https://github.com/openshift/installer/pull/5196/commits/d924c56f59ad915ba6f88b205fcf4f08267bf956#r769714102